content/page/events.en.md

```diff
@@ -8,6 +8,13 @@ draft: false
 Exodus Privacy is regulary performing talks about its works and results, in French or in English. If you want to invite us, feel free to [contact us](/en/page/who/).
 
 
+# Next events
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), INSA library, Rennes (France), 3th February 2020 
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), library of the University of Nantes (France), 11th February 2020 
```

Comment:
I think the link (and maybe the name also) is not correct

Reply:
The programmation of the 2020 edition is not published yet. So the link is the only one we have, and the name is ok.